src/common/snippets/src/lowered/pass/init_loops.cpp

```diff
@@ -56,7 +56,8 @@ bool InitLoops::run(LinearIR& linear_ir) {
         return false;
     }
 
-    const auto& loops = linear_ir.get_loop_manager()->get_map();
+    const lowered::LoopManagerPtr& loop_manager = linear_ir.get_loop_manager();
```

Comment:
Could you please explain why do we have to specify types in such cases? Is it clang-tidy limitation/bug?

Reply:
Regarding `LoopManagerPtr`: I noticed that this using is duplicated in linear_ir.hpp. Can it be the root cause of such clang-tidy behavior?

Reply:
That's a good idea. I have trierd it, but these things are not related. I have removed `using` for one case, but the problem when header is considered missing, but after removal compilation fails due to `error: member access into incomplete type 'element_type' (aka 'ov::snippets::lowered::LoopManager')` is still happening